src/schemas/transaction.yaml

```diff
@@ -1,3 +1,68 @@
+Transaction7702Unsigned:
+  type: object
+  title: EIP-7702 transaction.
+  required:
+    - type
+    - nonce
+    - gas
+    - value
+    - input
+    - maxFeePerGas
+    - maxPriorityFeePerGas
+    - gasPrice
+    - chainId
+    - accessList
+    - authorization list
+  properties:
+    type:
+      title: type
+      type: string
+      pattern: ^0x2$
+    nonce:
+      title: nonce
+      $ref: '#/components/schemas/uint'
+    to:
+      title: to address
+      oneOf:
+        - title: Contract Creation (null)
+          type: 'null'
+        - title: Address
+          $ref: '#/components/schemas/address'
+    gas:
+      title: gas limit
+      $ref: '#/components/schemas/uint'
+    value:
+      title: value
+      $ref: '#/components/schemas/uint'
+    input:
+      title: input data
+      $ref: '#/components/schemas/bytes'
+    maxPriorityFeePerGas:
+      title: max priority fee per gas
+      description: Maximum fee per gas the sender is willing to pay to miners in wei
+      $ref: '#/components/schemas/uint'
+    maxFeePerGas:
+      title: max fee per gas
+      description: The maximum total fee per gas the sender is willing to pay (includes the network / base fee and miner / priority fee) in wei
+      $ref: '#/components/schemas/uint'
+    gasPrice:
```

Comment:
this isn't a valid field for 7702 txs

Reply:
should I remove it I just copied eip 1559 txn and added authorizationlist as mentioned by matsee
thats why should I remove this field

Reply:
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7702.md

The EIP spec got updated:

"of the outer transaction follow the same semantics as [EIP-4844](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4844.md)."